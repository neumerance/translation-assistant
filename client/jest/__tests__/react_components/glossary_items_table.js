'use strict';

import JestSupportConfig from '../../support/config';
JestSupportConfig.setGlobals();

import React from 'react';
import GlossaryItemsTable from '../../../app/bundles/App/components/layouts/main_menu';
import renderer from 'react-test-renderer';

const router = {
  location: {
    pathname: {
      match: ()=>{ return true }
    }
  }
};

it('renders', () => {
  const component = renderer.create(
    <GlossaryItemsTable router={router}/>
  );

  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});
