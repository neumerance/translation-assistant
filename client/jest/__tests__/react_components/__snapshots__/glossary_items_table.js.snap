// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<nav
  className="navbar navbar-default navbar-static-top"
>
  <div
    className="container"
  >
    <div
      className="navbar-header"
    >
      <button
        aria-expanded="false"
        className="navbar-toggle collapsed"
        data-target="#bs-example-navbar-collapse-1"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="sr-only"
        >
          Toggle navigation
        </span>
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
      </button>
      <a
        className="navbar-brand"
        onClick={[Function]}
        style={Object {}}
      />
    </div>
    <div
      className="collapse navbar-collapse"
      id="bs-example-navbar-collapse-1"
    >
      <ul
        className="nav navbar-nav pull-right"
      >
        <li
          className="dropdown"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            className="dropdown-toggle"
            data-toggle="dropdown"
            href="#"
            role="button"
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "",
                }
              }
            />
            <span
              className="caret"
            />
          </a>
          <ul
            className="dropdown-menu pull-right"
          >
            <li>
              <a
                href="javascript:void(0)"
                onClick={[Function]}
              >
                Log Out
              </a>
            </li>
          </ul>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
